cypress/integration/analytics.test.js

```diff
@@ -161,13 +172,12 @@ describe('select_item event', () => {
 
     let skuId
 
-    cy.getById('product-link')
-      .first()
-      .within(() => {
-        cy.getById('buy-button').then(($btn) => {
-          skuId = $btn.attr('data-sku')
-        })
+    cy.getById('store-card').first().click()
+    cy.getById('buy-button')
```

Comment:
The select event is triggered when a product card is clicked, don't need to click on buy button.

Reply:
Indeed! I've removed the click to the buy button [here](https://github.com/vtex-sites/base.store/pull/284/commits/21462200625b0d4e5bd0504b70d76ca6ed214c52).